austation/code/modules/client/verbs/ooc.dm

```diff
@@ -1,5 +1,13 @@
+var/global/regex/is_blank_string_regex = new(@{"^(\s|[\u00A0\u2000\u2001\u2002\u2003\u2004\u2005\u2006\u2007\u2008\u2009\u200A\u2028\u205F\u3000])*$"})
+/proc/is_blank_string(var/txt)
+	if (is_blank_string_regex.Find(txt))
+		return 1
+	return 0 //not blank
+
 /client/ooc(msg as text)
 	. = ..()
+	if((length(msg) < 1) || is_blank_string(msg))
```

Comment:
```suggestion
	if((length(msg)) || is_blank_string(msg))
```

Reply:
Length returns the length of the string, not boolean

Reply:
in byond numbers can be true or false, in fact booleans don't actually exist. TRUE and FALSE are just defines for 1 and 0 respectively. However, since this isn't being used a boolean, I'd argue this makes more sense. And the suggestion is logically incorrect anyway. Should have been !(length(msg))